### PR TITLE
rename to pacta.multi.loanbook

### PR DESCRIPTION
**Pull request by cjyetman**
- relates to #121 (though should be followed by renaming the repo on GitHub)

### CODE REVIEW

**Comment by jdhoffa** [comment]
You'll probably need to change the name of [workflow.template.pacta.Rproj](https://github.com/RMI-PACTA/workflow.multi.loanbook/blob/main/workflow.template.pacta.Rproj) as well, no?

**Comment by jdhoffa** [comment]
Ahahah also just noticing it's called `workflow.template.pacta` and not `workflow.multi.loanbook`

**Review by jdhoffa — APPROVED** [review]

**Comment by jdhoffa** [comment]
@jacobvjk @cjyetman I will suggest that we merge this **after** #134 is merged, and then pause PRs while we change the name of the GH repo

**Review by jacobvjk — APPROVED** [review]

**Review by jdhoffa — APPROVED** [review]

**Pull request merged**
